packages/chart/src/Chart.tsx

```diff
@@ -531,6 +531,22 @@ class Chart extends Component<ChartProps, ChartState> {
         this.setState({ shownBlocker: null });
         break;
       }
+      case ChartModel.EVENT_TITLE_CHANGE: {
```

Comment:
Instead of an `EVENT_TITLE_CHANGE`, go with `EVENT_LAYOUT_UPDATED` and emit the new layout.
`ChartModel` already has the layout (`.getLayout()`) and that should contain the updated title.
Also see in `ChartModel.initData` how we use the chart layout on top of the state layout... so this would be something like:
```
case ChartModel.EVENT_LAYOUT_UPDATED: {
  const newLayout = detail as Partial<Layout>;
  this.setState(({ layout, revision }) => ({ layout: { ...layout, ...newLayout }, revision: revision + 1 }))
}
```